libraries/AC_AttitudeControl/AC_PosControl_Sub.cpp

```diff
@@ -38,5 +38,5 @@ void AC_PosControl_Sub::input_vel_accel_z(float &vel, const float accel, bool fo
         -accel_z_cms, accel_z_cms,
         _jerk_max_xy_cmsss, _dt, limit_output);
 
-    update_vel_accel(vel, accel, _dt, _limit_vector.z);
```

Comment:
this removal of limit_vector is intentional of course?

Reply:
Yes, this isn't done in copter so I think this was a copy past problem for sub.